src/components/Pathway/NodeGraph.js

```diff
@@ -99,20 +101,15 @@ function evaluateObj(obj) {
   return (new Function(` return  ${obj.split("=")[1]}` ));
 }
 
-const getAwarded = async(email) =>{
+const getAwarded = async(email) => {
   var resp = null;
 
   await axios({
       method: 'get',
-      url: `/award/user`,
-      data: {
-        body: {
-          email: email
-        }
-      }
+      url: `/award`,
   })
   .then(res => {
```

Comment:
Let's try to avoid using the promise callbacks. Prefer the async and await syntax.